packages/@aws-cdk/aws-stepfunctions/test/integ.intrinsics.ts

```diff
@@ -0,0 +1,56 @@
+import * as cdk from '@aws-cdk/core';
+import * as integ from '@aws-cdk/integ-tests';
+import { JsonPath, Pass, StateMachine } from '../lib';
+
+/*
+ * Stack verification steps:
+ *
+ * -- aws stepfunctions describe-state-machine --state-machine-arn <stack-output> has a status of `ACTIVE`
```

Comment:
This should use an assertion in code to verify instead of being added here in text.

Reply:
I created this integration test based on this.
I think it is not necessary to execute `describe-state-machine` explicitly to confirm that the state machine is deployed successfully because successful deployment of cfn stack means that state machine deployments are also succeeded, so I deleted the comment by https://github.com/aws/aws-cdk/pull/22431/commits/926c938b6df88fea14409133abfd714bfc1f9fc0.

https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-stepfunctions/test/integ.custom-state.ts

Reply:
Please add the assertion anyway. We didn't have that library when the older test was written but we do have that capability now.

Reply:
Ok, I added it. https://github.com/aws/aws-cdk/pull/22431/commits/d2ff6e1c2053915433f8a1aa5aed3219b766474d